managers/urls.py

```diff
@@ -8,14 +8,10 @@
     url("^projects/(?P<pk>[0-9]+)/$", views.ProjectDetailView.as_view(), name="custom-project-detail"),
     url("^projects/(?P<pk>[0-9]+)/update/$", views.ProjectUpdateView.as_view(), name="custom-project-update"),
     url("^projects/(?P<pk>[0-9]+)/delete/$", views.ProjectDeleteView.as_view(), name="custom-project-delete"),
+    url("^project/task-activities/$", views.ManageTaskActivities.as_view(), name="project-task-activities"),
     url(
-        "^project/(?P<pk>[0-9]+)/task-activities/$",
-        views.ManageTaskActivitiesInProjectView.as_view(),
-        name="project-task-activities",
-    ),
-    url(
-        "^project/(?P<pk>[0-9]+)/remove-task-activity/(?P<task_activity_pk>[0-9]+)$",
-        views.RemoveTaskActivityFromProjectView.as_view(),
+        "^project/remove-task-activity/(?P<task_activity_pk>[0-9]+)$",
+        views.RemoveTaskActivity.as_view(),
         name="remove-task-activity-from-project",
```

Comment:
It also should be changed. It's not `remove-task-activity-from-project` anymore :) It's not `remove-task-activity` or maybe even `disable-task-activity` because it real we don't delete it

Reply:
changed